lib/chef/provider/aws_cache_replication_group.rb

```diff
@@ -0,0 +1,45 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsCacheReplicationGroup < Chef::Provisioning::AWSDriver::AWSProvider
+
+  protected
+
+  def create_aws_object
+    converge_by "create new Elasticache Replication Group #{new_resource.name} in #{region}" do
+      new_resource.driver.elasticache.create_replication_group(desired_options)
+    end
+  end
+
+  def update_aws_object(cache_replication_group)
+    Chef::Log.warn('Updating Elasticache Replication Groups is currently unsupported')
+  end
```

Comment:
I think I'll change my statement after more thought and research - It's *possible* to get info about each cache cluster and do recursive updates but it's going to take a lot more logic and API calls. For the sake of getting basic support out there I think we should move forward with this PR and then come back around and add support for this and additional attributes.